### PR TITLE
Popup dialog, Ranked Label, fix UnitRestricts button, change default UnitCap option

### PR DESCRIPTION
**Pull request by Xinnony**
- Add Popup quick dialog (UIUtil)
- Add new Layout and the Layout file (LayoutHelpers)
- Add Ranked Label
- Fix and Optimize LobbyPreset UI, Lobby Options UI and Changelog UI (https://github.com/FAForever/fa/issues/272)
- Fix the "Unit Restricts" button (https://github.com/FAForever/fa/issues/323 https://github.com/FAForever/fa/issues/313)
- Separate Mod and Mod UI in Lobby Preset
- Change Unit Cap default (for Ranked settings)
- Changes "Score" and "Team Spawn" default options
- Set Ranked button set more ranked options
- Typo
- Convert all Tab to Space
- Change version to 2.6.1

Sheeo : Sorry is the last time i make large PR, next time i make a PR one by one.
Ps : I make always PNG file, but a devlopper can convert to DDS if he wants.

### CODE REVIEW

**Comment by Xinnony** [comment]
I let Sheeo or else do the last PR for the changelog.

**Comment by Xinnony** [comment]
https://github.com/FAForever/fa/issues/323
https://github.com/FAForever/fa/issues/313

**Comment by aeoncleanse** [comment]
Some of these were already fixed before this PR (#272), perhaps more

**Comment by Xinnony** [comment]
Yes sorry for layouthelpers.lua
The change is only :
https://github.com/FAForever/fa/pull/339/files#diff-2375e49494bdca037c33aa3a5159dcaeR93
https://github.com/FAForever/fa/pull/339/files#diff-2375e49494bdca037c33aa3a5159dcaeR107
https://github.com/FAForever/fa/pull/339/files#diff-2375e49494bdca037c33aa3a5159dcaeR471

File: lua/maui/layouthelpers.lua [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+end
+
+function AtRightTopIn(control, parent, rightOffset, topOffset)
+    rightOffset = rightOffset or 0
+    topOffset = topOffset or 0
+    control.Right:Set(function() return math.floor(parent.Right() - (rightOffset * pixelScaleFactor)) end)
+    control.Top:Set(function() return math.floor(parent.Top() + (topOffset * pixelScaleFactor)) end)
+end
+
+function AtRightBottomIn(control, parent, rightOffset, bottomOffset)
```
Comment (Xinnony) [inline lua/maui/layouthelpers.lua]: New function

File: lua/maui/layouthelpers.lua [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+end
+
+function AtLeftTopIn(control, parent, leftOffset, topOffset)
+    leftOffset = leftOffset or 0
+    topOffset = topOffset or 0
+    control.Left:Set(function() return math.floor(parent.Left() + (leftOffset * pixelScaleFactor)) end)
+    control.Top:Set(function() return math.floor(parent.Top() + (topOffset * pixelScaleFactor)) end)
+end
+
+function AtLeftBottomIn(control, parent, leftOffset, bottomOffset)
```
Comment (Xinnony) [inline lua/maui/layouthelpers.lua]: New function

File: lua/maui/layouthelpers.lua [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+    depth = depth or 1
+    control.Depth:Set(function() return parent.Depth() + depth end)
+end
+
+function DepthUnderParent(control, parent, depth)
+    depth = depth or 1
+    control.Depth:Set(function() return parent.Depth() - depth end)
+end
+
+function DepthSetTopOverParent(control, parent, depth)
```
Comment (Xinnony) [inline lua/maui/layouthelpers.lua]: New function

**Comment by Xinnony** [comment]
k.
I try to make an effort with you but receive 30 mail useless comment ...
I'm not very happy.

**Comment by Sheeo** [comment]
Closing this since I believe we're adressing all items in new pull requests, save for the popup dialog.

Future work on that refactor should be seperated from bugfixes.

**Comment by ChrisKitching** [comment]
There's also some cleanup for the preset system in here that'd be worth digging out, too.

**Comment by Sheeo** [comment]
I thought IceDreamer had revamped these?

**Pull request closed without merging**